### PR TITLE
Implement dump state

### PR DESCRIPTION
**Pull request by orkunkl**
Related to https://github.com/iov-one/weave/issues/1212

### CODE REVIEW

File: cmd/dumpstate/dumpstate.go [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+		var token username.Token
+		switch key, err := it.Next(store.CheckStore(), &token); {
+		case err == nil:
+			out = append(out, tokenFormat{
+				Username: string(key),
+				Targets:  token.Targets,
+				Owner:    token.Owner,
+			})
+		case errors.ErrIteratorDone.Is(err):
+			goto success
```
Comment (alpe) [inline cmd/dumpstate/dumpstate.go:152]: The use of `goto` is very uncommon in Go projects. Why not `return out, nil` here to exit early? You could also break with a label if you have to. 
Disclaimer: I did not review the PR. Just stumbled upon the `goto`.

**Comment by orkunkl** [comment]
- escrow addresses are also dumped as bech32
- contract signatures are dumped as hex
Are we good on these?

**Comment by davepuchyr** [comment]
> * escrow addresses are also dumped as bech32
> * contract signatures are dumped as hex
>   Are we good on these?

@orkunkl , Please use bech32 for all addresses.

**Comment by davepuchyr** [comment]
All escrow amounts are `null`.  Please investigate.

```
    {
      "source": "iov1v9pzqxpywk05xn2paf3nnsjlefsyn5xu3nwgph",
      "arbiter": "iov1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqvnwh0u",
      "destination": "iov1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqvnwh0u",
      "timeout": 1602331200,
      "amount": null,
      "address": "iov1ytn5f79hlu5y25phgwx79jmeqj78l6d2ey4s35"
    },
```

**Pull request merged**